### DISCUSSION TITLE
How do I build dependent packages?

### DISCUSSION THREAD

**Discussion by alekseykurylev** [vercel/turborepo#9140] [Help]
### Summary

Hi, I'm using the turborepo "with-tailwind" example. When I run the command `pnpm dev --filter=web`, the css file is not compiled in the "ui" package. How can I fix this?

https://github.com/vercel/turborepo/tree/main/examples/with-tailwind 

![Снимок экрана 2024-09-14 в 09 23 15](https://github.com/user-attachments/assets/b8ac4545-9215-4e96-9f8b-46acf5f1e77b)

**Comment by anthonyshew** [comment]
Hi, I'm unable to reproduce this with the example. Are you meaning that you started with the example and changed code from it?
